### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by christiam**

### CODE REVIEW

**Review by boratyng — DISMISSED** [review]

File: docs/source/exit-codes.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+==========
+
+=========  ============================================================================ ============================
+Exit code  Meaning                                                                      Applies to subcommand
+=========  ============================================================================ ============================
+0          Success                                                                      All
+1          Error in query sequence(s), BLAST options, or ElasticBLAST configuration     Submit
+2          Error in BLAST database                                                      Submit
+3          Error in BLAST engine                                                        Submit, status
+4          Out of memory                                                                Submit, status
```
Comment (boratyng) [inline docs/source/exit-codes.rst]: Is this supported?

File: docs/source/exit-codes.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+5          Timeout                                                                      Submit, status
+6          Error creating output files                                                  Submit, status
+7          Missing required dependency                                                  All
+8          Error communicating with cluster                                             All
+9          Operation interrupted                                                        All
+10         Search is in progress                                                        Status, if flag is provided
```
Comment (boratyng) [inline docs/source/exit-codes.rst]: I do not think this is implemented.

Comment (christiam) [reply docs/source/exit-codes.rst]: Removed

File: docs/source/exit-codes.rst (hunk 1 above)
Comment (christiam) [reply docs/source/exit-codes.rst]: It appears that isn't the case. I removed it.

File: docs/source/issues.rst [hunk 3]
```diff
@@ -18,9 +18,9 @@ Persistent disk not properly deleted
 ------------------------------------
 
 As part of its normal operation ElasticBLAST starts a
-:ref:`persistent disk <elb_pd_size>` and sometimes this is not properly
-deleted. To double check and delete it, please run the commands below
-accordingly:
+:ref:`persistent disk <elb_pd_size>` and under some circumstances this
```
Comment (boratyng) [inline docs/source/issues.rst]:
```suggestion
:ref:`persistent disk <elb_pd_size>` and under some circumstances it
```

**Review by boratyng — DISMISSED** [review]

**Pull request merged**